### ISSUE TITLE
Repo currently supported?

### ISSUE THREAD

**Issue by sabinewinkler** [fnproject/fdk-rust#2]
Hi,

great to find this experimental FDK rust implementation. Is this project still ongoing? Trying to init a rust function does not seem to be supported by the current fn version (fn version 0.5.29). Thanks for further information.

**Comment by zootalures** [comment]
no, sorry this is not actively maintained and will not work with recent versions of Fn due to FDK contract changes, I'll update the README accordingly.

**Comment by sabinewinkler** [comment]
Thanks for the quick reply. Do you know, if there is another project planned for rust fdk? Or will this one be activated later again?
